Users/Tutorial.rst

```diff
@@ -170,11 +170,14 @@ Thats all nice and well but we also have a Makefile for our project we
 want to check. So let us introduce another feature of our configuration
 syntax: *sections*.
 
-The line ``[Default]`` specifies that everything below will belong to
-the Default section. If nothing is specified, a setting will implicitly
-belong to this section.
+The line ``[cli]`` implies that everything below belongs to the special
```

Comment:
don't use cli over here since the user is not expected to create cli on his own. this may tempt them to do so. use all or any other name

Reply:
first explain the usage of sections using any arbitrary name and then explain that the section [cli] is the one created based on the command line arguments of the last coala run

Reply:
would be better if lines 321-327 is put over here

Reply:
explain everything about cli over here and be done with it

Reply:
Yes good idea @ishanSrt ! ;-) It will be much cleaner if we told them all about the `cli` section along with the lines 321-327 only. I'm on it. :+1:

Reply:
Hey @ishanSrt need your advice here:

While making the above mentioned changes, I realised that if we move the lines 321-327 here, it would break the narrative and create confusion in reader's mind. Those lines talk about the **Setting Inheritance** feature of coala- something which the user has not encountered yet in the tutorial. So, IMHO we shouldn't move them here.

What do you think I should do instead? 
One possible fix that I could think of was to move lines 173-177 over [here](https://github.com/coala/documentation/pull/537/files#diff-b9ec543d8e4599b29d0ecbfa344b9abaL160). This way we could still keep the narrative. But I'm not sure if that's the best possible solution to the problem.

Reply:
IMO you should put everything about cli in one place. but complete it first and maybe it turns out to be better. there is no right way to make someone understand stuff 👍